zh/deploy-on-gcp-gke.md

```diff
@@ -292,57 +291,39 @@ Check Table Before Drop: false
 
     ```hcl
     module "example-tidb-cluster" {
-    providers = {
-        helm = "helm.gke"
-    }
-    source                     = "../modules/gcp/tidb-cluster"
-    cluster_id                 = module.tidb-operator.cluster_id
-    tidb_operator_id           = module.tidb-operator.tidb_operator_id
-    gcp_project                = var.GCP_PROJECT
-    gke_cluster_location       = local.location
-    gke_cluster_name           = <gke-cluster-name>
-    cluster_name               = <example-tidb-cluster>
-    cluster_version            = "v3.0.1"
-    kubeconfig_path            = local.kubeconfig
-    tidb_cluster_chart_version = "v1.0.0"
-    pd_instance_type           = "n1-standard-1"
-    tikv_instance_type         = "n1-standard-4"
-    tidb_instance_type         = "n1-standard-2"
-    monitor_instance_type      = "n1-standard-1"
-    pd_node_count              = 1
-    tikv_node_count            = 2
-    tidb_node_count            = 1
-    monitor_node_count         = 1
+      providers = {
+          helm = "helm.gke"
+      }
+      source                     = "../modules/gcp/tidb-cluster"
+      cluster_id                 = module.tidb-operator.cluster_id
+      tidb_operator_id           = module.tidb-operator.tidb_operator_id
+      gcp_project                = var.GCP_PROJECT
+      gke_cluster_location       = local.location
+      gke_cluster_name           = <gke-cluster-name>
+      cluster_name               = <example-tidb-cluster>
+      cluster_version            = "v3.0.1"
+      kubeconfig_path            = local.kubeconfig
+      tidb_cluster_chart_version = "v1.0.0"
+      pd_instance_type           = "n1-standard-1"
+      tikv_instance_type         = "n1-standard-4"
+      tidb_instance_type         = "n1-standard-2"
+      monitor_instance_type      = "n1-standard-1"
+      pd_node_count              = 1
+      tikv_node_count            = 2
+      tidb_node_count            = 1
+      monitor_node_count         = 1
     }
     ```
 
     > **注意：**
     >
     > - 每个集群的 `cluster_name` 必须是唯一的。
-    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以该 Region 中可用区的个数。
-
-    你可以通过 `kubectl` 获取创建的 TiDB 集群和监控组件的地址。如果你希望 Terraform 脚本打印此信息，可在 `outputs.tf` 中添加一个 `output` 配置项，如下所示：
-
-    {{< copyable "" >}}
-
-    ```hcl
-    output "how_to_connect_to_example_tidb_cluster_from_bastion" {
-    value = module.example-tidb-cluster.how_to_connect_to_tidb_from_bastion
-    }
-    ```
-
-    上述配置可使该脚本打印出用于连接 TiDB 集群的命令。
+    > - 为任一组件实际创建的总节点数等于配置文件中的节点数乘以配置的可用区的个数（Regional 集群默认为 3）。
 
 2. 修改完成后，执行以下命令来创建集群。
 
     {{< copyable "shell-regular" >}}
 
-    ```bash
```

Comment:
This is required since we add a new module in the above step.

Reply:
done